### PR TITLE
get_schedule_by_regid_and_term has verbose on

### PR DESCRIPTION
**Pull request by fanglinfang**
Made get_schedule_by_regid_and_term always verbose-on.
Merged UnfinishedPceCourse (only used by MyUW) into Registration model.

### CODE REVIEW

File: uw_sws/enrollment.py [hunk 1]
```diff
@@ -151,23 +151,13 @@ def is_unfinished_pce_course(registration):
...
-    unf_pce_course.request_status = json_data['RequestStatus']
-    unf_pce_course.meta_data = json_data['Metadata']
-    return unf_pce_course
+def _json_to_section_ref(reg_json, term):
+    return SectionReference(
+        term=term,
+        curriculum_abbr=reg_json['Section']['CurriculumAbbreviation'],
+        course_number=reg_json['Section']['CourseNumber'],
+        section_id=reg_json['Section']['SectionID'],
+        url=reg_json['Section']['Href'])
```
Comment (ezturner) [inline uw_sws/enrollment.py]: Why aren't we using the UnfinishedPceCourse model here anymore?

Comment (fanglinfang) [reply uw_sws/enrollment.py]: The original data in the enrollments search response is Registrations. 
Now all the data and methods are available in the Registration model, this one should be eliminated.

File: uw_sws/models.py [hunk 2]
```diff
@@ -19,6 +20,12 @@
     "{year}-{quarter}-{curr_abbr}-{course_num}-{section_id}-{inst_regid}")
 
 
+def date_to_json(dt):
```
Comment (jlaney) [inline uw_sws/models.py]: This needs to be moved to uw_sws.util and renamed to date_to_str().

What was wrong with str(dt)?

Comment (fanglinfang) [reply uw_sws/models.py]: when the dt is None, str(None) returns 'None'. 
Does it matter?

**Pull request merged**